apps/wallet-mobile/src/features/Settings/WalletSettings/WalletSettingsScreen.tsx

```diff
@@ -26,100 +28,82 @@ const iconProps = {
 export const WalletSettingsScreen = () => {
   const intl = useIntl()
   const strings = useStrings()
-  const {navigation, resetToWalletSelection} = useWalletNavigation()
+  const {resetToWalletSelection} = useWalletNavigation()
   const wallet = useSelectedWallet()
-  const easyConfirmationEnabled = useEasyConfirmationEnabled(wallet)
   const authSetting = useAuthSetting()
   const logout = useLogout()
   const settingsNavigation = useNavigation<SettingsRouteNavigation>()
+  const easyConfirmationEnabled = useEasyConfirmationEnabled(wallet)
 
   const onSwitchWallet = () => {
     resetToWalletSelection()
   }
 
-  const onEnableEasyConfirmation = () => {
-    navigation.navigate('app-root', {
```

Comment:
now its going directly to the screen

Reply:
https://github.com/Emurgo/yoroi/pull/2676/files#diff-b0510212f2210c23d1f4777217f94d522d360f2d316cfa4070e5567d64249354R10